lib/constants.dart

```diff
@@ -1,182 +1,220 @@
+  2220537248
 ];
 const List<int> _envieddataaccessMessage = [
-  3062738999,
-  562173714,
-  3682346694,
-  3197932527,
-  633954907,
-  1384068281,
-  7155185,
-  603945793,
-  4148589975,
-  101904216,
-  4012612437,
-  2318603296,
-  3407770579,
-  195091435,
-  715795492,
-  1099583035,
-  1413788809,
-  282283929,
-  2768189149,
-  2348979180,
-  4121141208,
-  37434019,
-  478250765,
-  1579862763,
-  1474136350,
-  3316177948,
-  487431844,
-  1789076013,
-  3040220024,
-  85742283,
-  43880015,
-  140106688,
-  3234139069,
-  17839458,
-  252980201,
-  885062129,
-  3410170219,
-  3900724582,
-  2307485097,
-  3395375719,
-  663600709,
-  1605092441,
-  3391568632,
-  2032453917,
-  3898623084,
-  1380979023,
-  3166975038,
-  3655661322,
-  3270529604,
-  882170023,
-  1254761917,
-  2617879821,
-  524156346,
-  651786646,
-  3460538430,
-  3416530264,
-  1650517848,
-  1506068345,
-  1882813613,
-  2646652633,
-  1633865430,
-  3145159628,
-  2637239161,
-  4133742074,
-  3333179616,
-  3902421352,
-  2318813295,
-  1207510395,
-  1643915290,
-  1041562177,
-  2343570420,
-  1022649073,
-  1382610663,
-  366143689,
-  3547779164,
-  2412936740,
-  398341719,
-  985624048,
-  2475800059,
-  1336222443,
-  2134040705,
-  1732887085,
-  458103860,
-  1009671177,
-  1733240160,
-  1301538495,
-  962219034,
-  3719805989
+  3644477830,
+  877016796,
+  3190791897,
+  1075549304,
+  2045847567,
+  1939237649,
+  1719605493,
+  3836368947,
+  1494506247,
+  4006902126,
+  1814599228,
+  1484002248,
+  575279047,
+  3975252019,
+  2461063719,
+  1919876364,
+  3835005493,
+  1571540060,
+  2553792424,
+  1839179042,
+  417714649,
+  1435866317,
+  3718161283,
+  2277502438,
+  2961047163,
+  3900745264,
+  3403420221,
+  1783142548,
+  1761264247,
+  2384647719,
+  4106352391,
+  4135463751,
+  2898871512,
+  1110406024,
+  2470432449,
+  4249593318,
+  3025158697,
+  2333650881,
+  4115882896,
+  1894726059,
+  2955562908,
+  664643603,
+  964472780,
+  1610593125,
+  329266345,
+  3375201259,
+  2565877925,
+  388425615,
+  3992929585,
+  921941667,
+  396692350,
+  331826829,
+  3487213060,
+  1144105800,
+  1003246594,
+  3694421698,
+  3004597872,
+  868304975,
+  1806261005,
+  3191828134,
+  2183486302,
+  444991910,
+  1194654989,
+  3112079418,
+  2133756446,
+  2869130153,
+  4255384062,
+  3179988585,
+  2149463806,
+  3492777115,
+  3519144763,
+  212318906,
+  237720682,
+  3801606344,
+  1741331770,
+  270612568,
+  1363430049,
+  2889682807,
+  20179147,
+  1322612876,
+  1742692326,
+  372543209,
+  2311448095,
+  1098640270,
+  2735581809,
+  191019916,
+  1285642633,
+  37440447,
+  1378778573,
+  5604498,
+  2038122354,
+  1842702363,
+  1411569175,
+  446742627,
+  3246391097,
+  273560492,
+  3304366640,
+  2768127751,
+  2825130248,
+  2860707362,
+  4096782183,
+  1008677224,
+  3346798882,
+  320737591,
+  3686011434,
+  2514602120,
+  2220537230
 ];
 final String POLYGONID_ACCESS_MESSAGE = String.fromCharCodes(
```

Comment:
As discussed, the obfuscation of `POLYGONID_ACCESS_MESSAGE` is useless as it's deobfuscated here.

Reply:
removed access message